src/content/app/genome-browser/state/focus-object/focusObjectSlice.ts

```diff
@@ -130,17 +130,12 @@ const buildFocusVariantObject = (payload: {
   variantId: string;
 }): FocusVariant => {
   const { genomeId, objectId, variantId } = payload;
+
   return {
     type: 'variant',
     genome_id: genomeId,
     object_id: objectId,
-    label: variantId,
-    location: {
-      // just some arbitrary hardcoded location; this will change to proper data in the future
-      chromosome: '13',
-      start: 1_000_000,
-      end: 1_000_001
-    }
```

Comment:
`BasicFocusObject` type, from which the `FocusVariant` type extends, is defined as having a location. We should probably keep it.

Reply:
✅